src/MyClass.java

```diff
@@ -0,0 +1,8 @@
+public class MyClass {
+    public static void main(String[] args) {
+        int a = 2;
+        int b = 3;
+        int c = a + b;
+        System.out.println("sumOfNumbers" + " " + "=" + " " + c);
```

Comment:
to call some method it's needed to write sumOfNumbers(method_arguments_should_be _here):
 System.out.println(sumOfTwoNumbers(2, 5));